icommons_ext_tools/cron.d/prod/qualtrics_user_update_cron

```diff
@@ -1,2 +1,2 @@
 # Update all required Qualtrics users at 4 am every Sunday
-00 4 * * 0 deploy /var/opt/virtualenvs/icommons_ext_tools/bin/python /opt/tlt/icommons_ext_tools/manage.py qualtrics —perform-full-update >> /var/opt/tlt/logs/django-icommons_ext_tools.log 2>&1
+0 4 * * 0 deploy /home/deploy/.virtualenvs/icommons_ext_tools/bin/python /opt/django/icommons_ext_tools/manage.py qualtrics —perform-full-update >> /var/opt/django/log/django-icommons_ext_tools.log 2>&1
```

Comment:
I guess the 00-->0 shouldn't matter, as it would have been interpreted as 0 mins anyways..